2025-01-31 16:11:15,692 INFO    MainThread:21036 [wandb_setup.py:_flush():68] Current SDK version is 0.19.2
2025-01-31 16:11:15,692 INFO    MainThread:21036 [wandb_setup.py:_flush():68] Configure stats pid to 21036
2025-01-31 16:11:15,693 INFO    MainThread:21036 [wandb_setup.py:_flush():68] Loading settings from C:\Users\palig\.config\wandb\settings
2025-01-31 16:11:15,693 INFO    MainThread:21036 [wandb_setup.py:_flush():68] Loading settings from c:\Users\palig\MSE\Vertiefungsprojekt1\mse_vp1_rulestimation\notebooks\model_training\wandb\settings
2025-01-31 16:11:15,693 INFO    MainThread:21036 [wandb_setup.py:_flush():68] Loading settings from environment variables
2025-01-31 16:11:15,693 INFO    MainThread:21036 [wandb_init.py:_log_setup():523] Logging user logs to c:\Users\palig\MSE\Vertiefungsprojekt1\mse_vp1_rulestimation\notebooks\model_training\wandb\run-20250131_161115-corfzbkr\logs\debug.log
2025-01-31 16:11:15,695 INFO    MainThread:21036 [wandb_init.py:_log_setup():524] Logging internal logs to c:\Users\palig\MSE\Vertiefungsprojekt1\mse_vp1_rulestimation\notebooks\model_training\wandb\run-20250131_161115-corfzbkr\logs\debug-internal.log
2025-01-31 16:11:15,695 INFO    MainThread:21036 [wandb_init.py:_jupyter_setup():469] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F517C02C90>
2025-01-31 16:11:15,695 INFO    MainThread:21036 [wandb_init.py:init():641] calling init triggers
2025-01-31 16:11:15,695 INFO    MainThread:21036 [wandb_init.py:init():647] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.001, 'epochs': 20, 'model': 'ATT_13'}
2025-01-31 16:11:15,695 INFO    MainThread:21036 [wandb_init.py:init():674] starting backend
2025-01-31 16:11:15,911 INFO    MainThread:21036 [wandb_init.py:init():678] sending inform_init request
2025-01-31 16:11:15,925 INFO    MainThread:21036 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2025-01-31 16:11:15,926 INFO    MainThread:21036 [wandb_init.py:init():693] backend started and connected
2025-01-31 16:11:15,928 INFO    MainThread:21036 [wandb_run.py:_label_probe_notebook():1226] probe notebook
2025-01-31 16:11:15,935 INFO    MainThread:21036 [wandb_init.py:init():786] updated telemetry
2025-01-31 16:11:15,957 INFO    MainThread:21036 [wandb_init.py:init():818] communicating run to backend with 90.0 second timeout
2025-01-31 16:11:16,621 INFO    MainThread:21036 [wandb_init.py:init():868] starting run threads in backend
2025-01-31 16:11:16,731 INFO    MainThread:21036 [wandb_run.py:_console_start():2415] atexit reg
2025-01-31 16:11:16,731 INFO    MainThread:21036 [wandb_run.py:_redirect():2265] redirect: wrap_raw
2025-01-31 16:11:16,731 INFO    MainThread:21036 [wandb_run.py:_redirect():2330] Wrapping output streams.
2025-01-31 16:11:16,731 INFO    MainThread:21036 [wandb_run.py:_redirect():2355] Redirects installed.
2025-01-31 16:11:16,733 INFO    MainThread:21036 [wandb_init.py:init():910] run started, returning control to user process
2025-01-31 16:12:07,357 INFO    MainThread:21036 [wandb_run.py:_finish():2140] finishing run p-alig-fh-ost/mse-vp1-test/corfzbkr
2025-01-31 16:12:07,359 ERROR   MainThread:21036 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-01-31 16:12:07,359 INFO    MainThread:21036 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-31 16:12:07,359 INFO    MainThread:21036 [wandb_init.py:_jupyter_teardown():451] cleaning up jupyter logic
2025-01-31 16:12:07,359 INFO    MainThread:21036 [wandb_run.py:_atexit_cleanup():2380] got exitcode: 0
2025-01-31 16:12:07,359 INFO    MainThread:21036 [wandb_run.py:_restore():2362] restore
2025-01-31 16:12:07,359 INFO    MainThread:21036 [wandb_run.py:_restore():2368] restore done
2025-01-31 16:12:09,476 INFO    MainThread:21036 [wandb_run.py:_footer_history_summary_info():3984] rendering history
2025-01-31 16:12:09,477 INFO    MainThread:21036 [wandb_run.py:_footer_history_summary_info():4016] rendering summary
2025-01-31 16:12:09,483 INFO    MainThread:21036 [wandb_run.py:_footer_sync_info():3945] logging synced files
